hyperparameter_tuning/blazingtext_text_classification_dbpedia/hpo_blazingtext_text_classification_dbpedia.ipynb

```diff
@@ -0,0 +1,710 @@
+{
```

Comment:
<p>Not just cost - also because the compute power is not needed for inference like it is for training. Add a sentence on size and other instance differences between <code>ml.c4.4xlarge</code>  and <code>ml.m4.xlarge</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>